internal/pkg/runtime/engines/singularity/container.go

```diff
@@ -1588,6 +1591,7 @@ func (c *container) addLibsMount(system *mount.System) error {
 		sessionFilePath, _ := c.session.GetPath(sessionFile)
 
 		err := system.Points.AddBind(mount.FilesTag, lib, sessionFilePath, flags)
+		sylog.Verbosef("Default mount: /libs:/libs")
```

Comment:
Correct me if I'm wrong, it seems like `/lib` is not a default mount point with the default singularity config file...
*If thats the case, I should probably remove this line.*

Reply:
@WestleyK Right, this code is reached when used with `--nv` and the final destination is `/.singularity.d/libs`